traffic-sniffer/lib/request-metadata/setup-file.ts

```diff
@@ -0,0 +1,59 @@
+import fs from 'fs'
+import { SnifferConfig } from './sniffer/sniffer';
+import { SnifferManager } from '../..';
+
+const setupFilePath = '../config/sniffers-setup.json'
```

Comment:
Add an option to receive the file path from environment variables.
Also the default path should be where the process starts.